src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -460,34 +460,104 @@ public FeedbackSessionQuestionsBundle getFeedbackSessionQuestionsForStudent(
         InstructorAttributes instructorGiver = null;
         StudentAttributes studentGiver = student;
 
-        
-        updateBundleAndRecipientListWithResponses(userEmail, student,
+        Set<String> hiddenInstructorEmails = getHiddenInstrutorEmails(courseId);
+        updateBundleAndRecipientListWithResponsesForStudent(userEmail, student,
                 bundle, recipientList, question, instructorGiver,
-                studentGiver);
+                studentGiver, hiddenInstructorEmails);
         
 
         return new FeedbackSessionQuestionsBundle(fsa, bundle, recipientList);
     }
 
-    private void updateBundleAndRecipientListWithResponses(
+    private void updateBundleAndRecipientListWithResponsesForStudent(
             String userEmail,
             StudentAttributes student,
             Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> bundle,
             Map<String, Map<String, String>> recipientList,
             FeedbackQuestionAttributes question,
-            InstructorAttributes instructorGiver, StudentAttributes studentGiver)
+            InstructorAttributes instructorGiver, StudentAttributes studentGiver,
```

Comment:
Yes, the `instructorGiver` parameter can be removed. I will remove it in the next few commits.